fusor-ember-cli/app/components/text-f.js

```diff
@@ -40,6 +40,12 @@ export default Ember.Component.extend({
     }
   }),
 
+  invalidNetworkRange: Ember.computed('value', 'isNetworkRange', function() {
+      // TODO
+      return false;
```

Comment:
added placeholder for validating network range.
@jmontleon, what is the regex?  Is it a valid IP addresss plus \/d+